### PR TITLE
Fallback Provider

### LINKED ISSUE #1628: Rollout fallback provider to testnet3
Rollout fallback provider on testnet3 scraper, validator and relayer agents

### PR DESCRIPTION
**Pull request by mattiecnvr**
### Description

Minor change to configs to support http fallback and then tested against testnet3 in rc and deployed.

### Drive by Changes
- Updated hello world rc image for mainnet2 and testnet3

### Related Issues
- Fixes #1628

### CODE REVIEW

**Comment by mattiecnvr** [comment]
Logs showing it working in rc
https://cloudlogging.app.goo.gl/WuwBCp19ME7aQxYy9

File: typescript/infra/config/environments/mainnet2/helloworld.ts [hunk 1]
```diff
@@ -33,7 +33,7 @@ export const releaseCandidate: HelloWorldConfig<MainnetChains> = {
   kathy: {
     docker: {
       repo: 'gcr.io/abacus-labs-dev/hyperlane-monorepo',
-      tag: 'sha-19dc142',
+      tag: 'sha-c7cbd2b',
```
Comment (asaj) [inline typescript/infra/config/environments/mainnet2/helloworld.ts:36]: Include in "drivebys"

File: typescript/infra/config/environments/testnet3/helloworld.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ export const releaseCandidate: HelloWorldConfig<TestnetChains> = {
   kathy: {
     docker: {
       repo: 'gcr.io/abacus-labs-dev/hyperlane-monorepo',
-      tag: 'sha-6ee34e4',
+      tag: 'sha-c7cbd2b',
```
Comment (asaj) [inline typescript/infra/config/environments/testnet3/helloworld.ts:35]: Ditto

**Review by asaj — APPROVED** [review]
